tests/plugins/inlinetoolbar/manual/focus.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
```

Comment:
This markdown file has a wrong formatting, just look how it looks like:

![test descirption rendered](https://i.imgur.com/4qV18PE.png)